views/sidecolumn.html

```diff
@@ -1,36 +1,91 @@
 <!-- Filters Section -->
 {{ define "sidecolumn" }}
-<div id="sidecolumn" class="mt-12 ml-12">
-    <p>{{ .Count }} Results</p>
-    <div class="flex flex-col space-y-2">
-        {{ range .Filters }}
-        <div class="flex items-center space-x-2">
-            <input
-                    {{ if .Active }} checked {{ end }}
-                    type="checkbox"
-                    name="{{ .Name }}"
-                    id="{{ .Name }}"
-                    hx-post="/filter/{{.ID}}"
-                    hx-swap="innerHTML"
-                    hx-target="#results-container"
-                    hx-trigger="click"
-                    hx-select-oob="#sidecolumn"
-            >
-            <label for="{{ .Name }}" class="text-gray-700">{{ .Name }}</label>
+<aside id="sidecolumn" class="mx-6 mt-6 w-64 rounded-md bg-white">
+    {{ range $index, $filter := .Filters }}
+    <div id="accordion-{{ $index }}" class="border-b border-gray-200">
```

Comment:
currently the button does not fill the entire div, so there is space where i cannot click. perhaps you can fix this but making the button the top most layer or making entire div clickable.